asgs install fails on qbc due to gnuplot build failure when trying to compile qtterminal

Attempted install on qbc failed with the following message
```
depbase=`echo qtterminal/qt_term.o | sed 's|[^/]*$|.deps/&|;s|\.o$||'`;\
c++ -DHAVE_CONFIG_H -I. -I..  -I../term -I../term -DBINDIR=\"/work/jgflemin/opt/default/bin\" -DX11_DRIVER_DIR=\"/work/jgflemin/opt/default/libexec/gnuplot/5.4\" -DQT_DRIVER_DIR=\"/work/jgflemin/opt/default/libexec/gnuplot/5.4\" -DGNUPLOT_SHARE_DIR=\"/work/jgflemin/opt/default/share/gnuplot/5.4\" -DGNUPLOT_PS_DIR=\"/work/jgflemin/opt/default/share/gnuplot/5.4/PostScript\" -DGNUPLOT_JS_DIR=\"/work/jgflemin/opt/default/share/gnuplot/5.4/js\" -DGNUPLOT_LUA_DIR=\"/work/jgflemin/opt/default/share/gnuplot/5.4/lua\" -DCONTACT=\"gnuplot-bugs@lists.sourceforge.net\" -DHELPFILE=\"/work/jgflemin/opt/default/share/gnuplot/5.4/gnuplot.gih\" -DGNUPLOT_X11=\"`echo gnuplot_x11 | sed 's,x,x,'`\" -DXAPPLRESDIR=\"@X11_APPDEFAULTS_DIR@\" -DGNUPLOT_QT=\"`echo gnuplot_qt | sed 's,x,x,'`\" -DQTGNUPLOT_DATA_DIR=\"/work/jgflemin/opt/default/share/gnuplot/5.4/qt\" -I/work/jgflemin/opt/default/include  -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng15 -I/usr/include/uuid -I/usr/include/libdrm -I/usr/include/pango-1.0 -I/usr/include/fribidi -I/usr/include/harfbuzz   -I/usr/include/qt5/QtCore -I/usr/include/qt5 -I/usr/include/qt5/QtGui -I/usr/include/qt5/QtNetwork -I/usr/include/qt5/QtSvg -I/usr/include/qt5/QtWidgets -I/usr/include/qt5/QtPrintSupport    -g -O2 -fPIC  -MT qtterminal/qt_term.o -MD -MP -MF $depbase.Tpo -c -o qtterminal/qt_term.o qtterminal/qt_term.cpp &&\
mv -f $depbase.Tpo $depbase.Po
In file included from /usr/include/c++/4.8.2/type_traits:35:0,
                 from /usr/include/qt5/QtCore/qglobal.h:45,
                 from /usr/include/qt5/QtCore/QtCore:4,
                 from qtterminal/qt_term.cpp:51:
/usr/include/c++/4.8.2/bits/c++0x_warning.h:32:2: error: #error This file requires compiler and library support for the ISO C++ 2011 standard. This support is currently experimental, and must be enabled with the -std=c++11 or -std=gnu++11 compiler options.
 #error This file requires compiler and library support for the \
  ^
```
Followed by a bunch of additional error messages, ending with the following:
```
make[3]: *** [qtterminal/qt_term.o] Error 1
make[3]: Leaving directory `/tmp/jgflemin-asgs/gnuplot-5.4.1/src'
make[2]: *** [install-recursive] Error 1
make[2]: Leaving directory `/tmp/jgflemin-asgs/gnuplot-5.4.1/src'
make[1]: *** [install] Error 2
make[1]: Leaving directory `/tmp/jgflemin-asgs/gnuplot-5.4.1/src'
make: *** [install-recursive] Error 1
post condition for "Step for installing gnuplot" FAILED, stopping. Please fix and rerun.
```
My suggestion is to turn off the build of qtterminal (it does not seem to be needed since the reason for including gnuplot is to create little prototypical plots of things in an automated way).

--- a/cloud/general/init-gnuplot-noX11.sh
+++ b/cloud/general/init-gnuplot-noX11.sh
@@ -32,7 +32,7 @@ rm -rf ./$GNUP_DIR 2> /dev/null
 tar zxvf ./$GNUP_TGZ
 
 cd $GNUP_DIR
-./configure --prefix=$OPT
+./configure --prefix=$OPT --without-qt --without-cairo --disable-wxwidgets
 make -j $JOBS
 make install
 

